lib/api-list-data.js

```diff
@@ -194,13 +194,18 @@ module.exports = class ApiListData extends API {
 			if(this.isValidObject(formattedFilters))
 				getParams.filters = formattedFilters;
 
+			this.shouldReturnOnlyTotal = this.shouldCalculateOnlyTotals();
```

Comment:
Como puse en otros comentarios anteriores, solo tendria 1 variable `shouldReturnTotals` 
```suggestion
			this.shouldReturnTotal = this.shouldCalculateTotals() || this.shouldCalculateOnlyTotals();
```